rosdep/python.yaml

```diff
@@ -620,6 +622,7 @@ python-defusedxml:
   debian: [python-defusedxml]
   fedora: [python-defusedxml]
   gentoo: [dev-python/defusedxml]
+  opensuse: [python3-defusedxml]
```

Comment:
It seems not in the default installation, I don't see it in an official repo(http://software.opensuse.org/package/python-defusedxml).  And now I remember I have another repo(devel:language:python3) being added.
Yeah, I don't see instructions for openSUSE,too.  I start from the "Source Installation" that for Ubuntu and Debian, and thought that would possible for openSUSE since they are all Linux distorts :P